Dockerfile

```diff
@@ -114,14 +114,19 @@ ENV NODE_ENV ${NODE_ENV}
 # Install wasm-bindgen-cli in the same profile as other components, to sacrifice some performance & disk space to gain
 # better build caching
 WORKDIR /platform
+# TODO: refactor to not need the wasm-bindgen-cli and remove the copy below, as deps stage should be independent
+COPY Cargo.lock .
```

Comment:
that will invalidate layer cache pretty often. maybe we just stick on specific version of this library?

Reply:
Agreed to revert changes in Dockerfile